gpd/gpd-pocket/default.nix

```diff
@@ -0,0 +1,87 @@
+{ lib, pkgs, ... }:
+
+# GPD Pocket support
+#
+# Most of the configuration comes from the Arch Linux Wiki:
+# https://wiki.archlinux.org/index.php/GPD_Pocket
+#
+# Missing:
+# - media keys
+# - brightness control
+# - video over USB-C
+
+{
+  imports = [
+    ../../common/cpu/intel
+    ../../common/pc/laptop
+  ];
+
+  # kernel
+  # TODO: use https://github.com/jwrdegoede/linux-sunxi?
+  boot = {
+    kernelParams = ["fbcon=rotate:1"];
+    kernelPackages = pkgs.linuxPackages_4_14;
+  };
+
+  # wifi
+  nixpkgs.config.allowUnfree = true;
+  hardware.firmware = with pkgs; [
+    brcmfmac4356-firmware
+  ];
+
+  # power management
+  services.tlp = {
+    enable = true;
+    extraConfig = ''
+      DISK_DEVICES="mmcblk0"
+      DISK_IOSCHED="deadline"
+    '';
+  };
+
+  hardware.pulseaudio = {
+    enable = true;
+    extraConfig = ''
+      set-card-profile alsa_card.platform-cht-bsw-rt5645 HiFi
+      set-default-sink alsa_output.platform-cht-bsw-rt5645.HiFi__hw_chtrt5645_0__sink
+      set-sink-port alsa_output.platform-cht-bsw-rt5645.HiFi__hw_chtrt5645_0__sink [Out] Speaker
+    '';
+    daemon.config = {
+      realtime-scheduling = "no";
+    };
+  };
+
+  # TODO: fix media keys
+  # sound.mediaKeys.enable = true;
+
+  services.xserver = {
+    dpi = 168;
+    deviceSection = ''
+      Option      "AccelMethod"     "sna"
+      Option      "TearFree"        "true"
+      Option      "DRI"             "3"
+    '';
+    xrandrHeads = [ { output= "DSI1"; monitorConfig = "Option \"Rotate\" \"right\""; } ];
+    libinput = {
+      enable = true;
+    };
+    inputClassSections = [''
+      Identifier  "calibration"
+      MatchProduct    "Goodix Capacitive TouchScreen"
+      Option  "TransformationMatrix" "0 1 0 -1 0 1 0 0 1"
+    ''
+    ''
+      Identifier "GPD trackpoint"
+      MatchProduct "SINO WEALTH Gaming Keyboard"
+      MatchIsPointer "on"
+      Driver "libinput"
+      Option "MiddleEmulation" "1"
+      Option "ScrollButton" "3"
+      Option "ScrollMethod" "button"
+    ''];
+  };
+
+  environment.variables = {
+    GDK_SCALE = "2";         # DPI
```

Comment:
All non-mergable attributes should be set via `lib.mkDefault` so that they can be overrided in user profile:

```
GDK_SCALE = lib.mkDefault "2";
```